scripts/install_custom_dc.sh

```diff
@@ -83,8 +83,32 @@ terraform init && terraform apply \
   -var="project_id=$PROJECT_ID" \
   -var="dc_website_domain=$DOMAIN" -auto-approve
 
-echo "Successfully launched the installer in $PROJECT_ID."
-echo "Please don't forget to email custom-datacommons-support@google.com for data access."
-echo "Please also make sure to click on the activation email for the newly created domain."
-echo "You should expect the instance to be accessible via https://$PROJECT_ID-datacommons.com"
-echo "within 30 minutes or so after custom-datacommons-support@google.com responds."
+_success_msg="
+###############################################################################
+# Status: Successfully launched the installer in $PROJECT_ID.
+###############################################################################
+
+
+###############################################################################
+# Action required:
+Please don't forget to email support+custom@datacommons.org with your
+GCP project id for data access.
+###############################################################################
+
+
+###############################################################################
+# Action required:
+Please also make sure to click on the activation email for $DOMAIN
+If the contact email has previously been used to verify domains,
+then $DOMAIN will be already active without needing activation emails.
```

Comment:
Is this the same as saying: "This step is only required for new and unverified domains"?
The "Action required" heading is also optional, in this case.